decode.go

```diff
@@ -1252,13 +1252,19 @@ func unquoteBytes(s []byte) (t []byte, ok bool) {
 					if dec := utf16.DecodeRune(rr, rr1); dec != unicode.ReplacementChar {
 						// A valid pair; consume.
 						r += 6
-						w += utf8.EncodeRune(b[w:], dec)
+						// Do not replace with unescaped rune, need to follow reference behaviour,
+						// see https://github.com/nspcc-dev/neo-go/pull/2174.
```

Comment:
It's not correct. We need to unescape strings to get correct ones and C# sure does the same thing.